projects/gui/src/newtournamentdialog.cpp

```diff
@@ -83,6 +84,9 @@ NewTournamentDialog::NewTournamentDialog(EngineManager* engineManager,
 	connect(ui->m_browsePgnoutBtn, SIGNAL(clicked()),
 		this, SLOT(browsePgnout()));
 
+	connect(ui->m_browseFilesBtn, SIGNAL(clicked()),
```

Comment:
I think "m_browseFilesBtn" and "browseFiles()" are a bit too generic as names when they are only meant to browse opening suites. Maybe "m_browseOpeningSuiteBtn" and "browseOpeningSuite()"?